tests/HeliumExporterTest.php

```diff
@@ -47,7 +47,10 @@ public function testHelium() {
     if ($data) {
       // EXPERIMENTS - get germplasm and traits used.
 
-      // Load experiment active in raw phenotypes.
+      // Load experiment active in raw phenotypes for user id #1.
+      global $user;
+      $user->uid = 1;
```

Comment:
```suggestion
```
You never want to assign something to the global user account or there can be crazy side effects ;-p instead load the user you want like I did above.